src/applications/personalization/profile-2/tests/components/Profile.data-unavailable-error.unit.spec.js

```diff
@@ -93,6 +93,32 @@ async function errorAppearsOnAllPages(
   });
 }
 
+async function hideErrorNonVet(
+  pageNames = [
+    PROFILE_PATH_NAMES.MILITARY_INFORMATION,
+    PROFILE_PATH_NAMES.DIRECT_DEPOSIT,
+    PROFILE_PATH_NAMES.ACCOUNT_SECURITY,
+    PROFILE_PATH_NAMES.CONNECTED_APPLICATIONS,
+  ],
+) {
+  const initialState = createBasicInitialState();
+  initialState.user.profile.veteranStatus = null;
+
+  const view = render(<Profile2Router isLOA3 isInMVI />, {
+    initialState,
+  });
+  const spinner = view.queryByRole('progressbar');
+  await waitForElementToBeRemoved(spinner);
+
+  const alert = await view.queryByTestId(ALERT_ID);
```

Comment:
should be no need to `await` on `queryBy...`. `await` is just used for `findBy` queries, I believe.

Reply:
you were right! updated! 🎉